src/main/java/me/tomsdevsn/hetznercloud/objects/request/PlacementGroupAddServerRequest.java

```diff
@@ -1,13 +1,11 @@
 package me.tomsdevsn.hetznercloud.objects.request;
 
-import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 
 @Getter
 @AllArgsConstructor
-@JsonInclude(JsonInclude.Include.NON_NULL)
 public class PlacementGroupAddServerRequest {
     @JsonProperty("placement_group")
     private long placementGroup;
```

Comment:
Please use the `Long` class instead of data type `long` (capitalize `L`). Using the data type made some weird issues in the past.

Reply:
I think this got over-written with a force-push :D

Reply:
Oh not just the non-null attributes, but also the data type :smile:

Reply:
it has been a long day, I should stop for today